specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,1430 @@
+{
```

Comment:
Semantic diff to 2022-03-01-preview in main: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvbWFpbi9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvYW5hbHl6ZWNvbnZlcnNhdGlvbnMuanNvbiIsInRhcmdldFVybCI6Imh0dHBzOi8vcmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbS9BenVyZS9henVyZS1yZXN0LWFwaS1zcGVjcy85OTIwOTlkZmJkNTJmZjVlYzcwMGFjZjg5ZDlmNzhkNzliNjE5OGNlL3NwZWNpZmljYXRpb24vY29nbml0aXZlc2VydmljZXMvZGF0YS1wbGFuZS9MYW5ndWFnZS9wcmV2aWV3LzIwMjItMDUtMDEtcHJldmlldy9hbmFseXplY29udmVyc2F0aW9ucy5qc29uIn0=

Reply:
No changes were detected.